pkg/framework/context/context.go

```diff
@@ -79,8 +81,13 @@ func (p *Provider) Service(id string) (interface{}, error) {
 	return nil, api.ErrSvcNotFound
 }
 
-// LegacyKMS returns a kms service.
+// LegacyKMS returns a legacyKMS service.
 func (p *Provider) LegacyKMS() legacykms.KeyManager {
+	return p.legacyKMS
+}
+
+// KMS returns a KMS service
```

Comment:
We could at least expand the acronym in these comments :).

Reply:
done